asistente_ladm_col/utils/qgis_utils.py

```diff
@@ -1395,3 +1395,9 @@ def show_help(self, module='', offline=False):
             url = web_url
 
         webbrowser.open("{}/{}".format(url, section))
+    
+    def clean_url(self, url):
```

Comment:
Please pass the function to `qt_utils`, call it `normalize_local_url` and redefine it simply as:

`return url[1:] if url.startswith("/") else url`

Reply:
This solve with commit: f320357faa16397b6e4e6a3276d8cf28ae2f21fa 
 Thanks for comment, solution it's nice.